DC_TTS_test.ipynb

```diff
@@ -0,0 +1,244 @@
+        "\n",
+        "orig_2 = 'logdir = \"logdir/LJ01\"'\n",
+        "repl_2 = 'logdir = \"/content/dc_tts/logdir/LJ01\"'\n",
+        "\n",
+        "orig_3 = \"sampledir = 'samples'\"\n",
+        "repl_3 = \"sampledir = '/content/dc_tts/samples'\"\n",
+        "file = \"/content/dc_tts/hyperparams.py\"\n",
+        "\n",
+        "replacement(file, orig_1, repl_1)\n",
+        "replacement(file, orig_2, repl_2)\n",
+        "replacement(file, orig_3, repl_3)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Cgqxu11lx8yk"
+      },
+      "source": [
+        "# download pre-trained model\n",
+        "import urllib.request\n",
+        "\n",
+        "url = \"https://www.dropbox.com/s/1oyipstjxh2n5wo/LJ_logdir.tar?dl=1\"  \n",
+        "u = urllib.request.urlopen(url)\n",
+        "data = u.read()\n",
+        "u.close()\n",
+        "with open('/content/dc_tts/LJ_logdir.tar', \"wb\") as f :\n",
+        "    f.write(data)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "j1pHd1Ww--3Y"
+      },
+      "source": [
+        "# extract the pre-trained model\n",
+        "import zipfile\n",
+        "with zipfile.ZipFile('/content/dc_tts/LJ_logdir.tar', 'r') as zip_ref:\n",
+        "    zip_ref.extractall('/content/dc_tts/')"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "qlfyO1BsgqbR"
+      },
+      "source": [
+        "# move the pre-trained model data to logdir folder\n",
+        "!mv /content/dc_tts/LJ01-1 /content/dc_tts/logdir\n",
+        "!mv /content/dc_tts/LJ01-2 /content/dc_tts/logdir"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "1Lt45CVjapZi"
+      },
+      "source": [
+        "# function to use own sentences for TTS\n",
+        "# with this, you can add as many sentences as you wish for the synthesis\n",
+        "def sentences_to_synthesize():\n",
+        "  x = input(\"Do you want to use the example sentences or synthesize your own sentences? Please choose(example/own): \")\n",
+        "  if x == \"example\":\n",
+        "    print(\"Good. Just go to the next cell then.\")\n",
+        "    pass\n",
+        "  elif x == \"own\":\n",
+        "    n = 1\n",
+        "    with open('/content/dc_tts/harvard_sentences.txt', 'w') as f:\n",
+        "      f.write('The code ignores the first line in the .txt. This is just a line so that everything works \\n')\n",
+        "\n",
+        "    own_sentence = input(\"Alright. Please write your sentence here: \")\n",
+        "    own_sentence = f\"{n}. {own_sentence} \\n\"\n",
+        "    \n",
+        "    with open('/content/dc_tts/harvard_sentences.txt', 'a') as f:\n",
```

Comment:
This looks like a very user-friendly idea!
Maybe, instead of overwriting the harvard sentences file, you could add an additional file.

Reply:
Good call, we now changed it so that if the user chooses to synthesize own sentences, it chooses own_sentences.txt and leaves harvard_sentences.txt unchanged.

Reply:
Simple yet effective implementation, does what it is supposed to do. Well done!